Packs/Slack/Integrations/Slack/Slack.yml

```diff
@@ -223,7 +223,7 @@ script:
     name: mirror-investigation
   - arguments:
     - default: true
-      description: The message content.
+      description: "The message content. When mentioning another slack user, make sure to do so in the following format: <@user_name>."
```

Comment:
Not sure about putting this in the argument description, the integration doc might be more appropriate. 
Also, this can also apply for tagging channels, e.g. #general etc

Reply:
I can add it there in addition, but when a customer is using the command I think that the argument description is more relevant than the readme, I'm afraid the customer will miss it if it is present only in the readme

Reply:
yeah in any case it should be added to the doc. and don't forget to add an example for a channel too :)